baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/MoreMatchers.java

```diff
@@ -74,6 +76,19 @@ static <T extends ClassTree> Matcher<T> classEnclosingClass(Matcher<ClassTree> m
         };
     }
 
+    static <T extends Tree> Matcher<T> hasAnnotationWithPackagePrefix(String prefix) {
+        return (classTree, state) -> {
+            Name packageName = state.getName(prefix);
+            Symbol sym = ASTHelpers.getSymbol(classTree);
+            for (Compound a : sym.getRawAttributes()) {
+                if (a.type.tsym.flatName().startsWith(packageName)) {
+                    return true;
+                }
+            }
+            return false;
+        };
+    }
```

Comment:
I think this is unused, only `SymbolPredicates.hasAnnotationWithPackage` appears to be referenced

Reply:
Yup, latent code.